CraftTweaker2-MC1120-Mod-JEI/src/main/resources/META-INF/crafttweaker_at.cfg

```diff
@@ -15,4 +15,5 @@ public net.minecraft.entity.projectile.EntityFishHook$State
 public net.minecraft.entity.projectile.EntityArrow field_70256_ap #knockbackStrength
 public-f net.minecraft.world.Explosion field_77281_g # affectedBlockPositions
 public net.minecraft.world.Explosion field_77287_j # world
-public net.minecraft.tileentity.MobSpawnerBaseLogic func_190895_g()Lnet/minecraft/util/ResourceLocation; # getEntityId
\ No newline at end of file
+public net.minecraft.tileentity.MobSpawnerBaseLogic func_190895_g()Lnet/minecraft/util/ResourceLocation; # getEntityId
+public net.minecraft.util.text.Style func_150224_n()Lnet/minecraft/util/text/Style; # getParent
```

Comment:
Any reason for the trailing `;`?

Reply:
Is the `;` useless? I don't know. I just copied from mcp bot.

Reply:
You know, I'm not actually sure, I thought it was since the other lines don't have it.

But if the bot gave it to you, then I would keep it